src/reference/asciidoc/amqp.adoc

```diff
@@ -3220,6 +3220,10 @@ Also, you must not have registered your own `ReturnCallback` with the `RabbitTem
 
 Starting with version 2.1.2, a method `replyTimedOut` has been added, allowing subclasses to be informed of the timeout so they can clean up any retained state.
 
+Starting with version 2.1.3, when using the default `DirectReplyToMessageListenerContainer`, you can add an error handler by setting the template's `replyErrorHandler` property.
```

Comment:
Do you mean `2.0.11` ?

I think during back-port to 2.0.x I'm going to ditch a What's New change, but leave this sentence here anyway.

Reply:
ok